native/lambda_game_engine/src/game.rs

```diff
@@ -540,3 +558,29 @@ fn apply_zone_effects(
         });
     }
 }
+
+fn nearest_player(
+    players: &Vec<&mut Player>,
+    position: &Position,
+) -> Option<(u64, Position)> {
+    let mut nearest_player = None;
+    let mut nearest_distance = 3000.0;
```

Comment:
can we tweak this down a bit? When playtesting, I noticed that it attacks in a direction even when there are no enemies within the newport

Reply:
This attribute is now configurable